docs/en/01-how-to-build/jetsons.md

```diff
@@ -286,6 +300,15 @@ make -j$(nproc) && make install
 It takes about 9 minutes to build SDK libraries on a Jetson Nano. So, please be patient until the installation is complete.
 ```
 
+If met following error during installation, please run command `apt-get install -y --no-install-recommends python3.9-dev` first.
```

Comment:
why need to install python3.9-dev? You are using python36 based on this command `pip3 install onnxruntime_gpu-1.10.0-cp36-cp36m-linux_aarch64.whl`.

Reply:
I'm not sure, I met the problem and found the solution in https://zhuanlan.zhihu.com/p/632183060
Seems the problem is common in Jetson.

Reply:
I can remove it if not necessary.

Reply:
personally, could remove it

Reply:
Done.